extensions/amp-date-picker/0.1/amp-date-picker.css

```diff
@@ -17,15 +17,31 @@
 @import '../../../third_party/react-dates/_datepicker.css';
 
 
-amp-date-picker,
-.SingleDatePicker_picker,
-.DateRangePicker_picker {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+amp-date-picker.i-amphtml-element {
```

Comment:
I don't think `i-amphtml-element` part is needed. This CSS loads at the same time that the component gets the `i-amphtml-element` class.

Reply:
good point, I'll remove the rule